fizz-buzz-streams/src/test/java/com/pubsap/eng/streams/FizzBuzzAggregatorTest.java

```diff
@@ -0,0 +1,191 @@
+package com.pubsap.eng.streams;
+
+import com.pubsap.eng.schema.*;
+
+import com.typesafe.config.Config;
+import com.typesafe.config.ConfigFactory;
+import io.confluent.kafka.schemaregistry.client.MockSchemaRegistryClient;
+import io.confluent.kafka.schemaregistry.client.rest.exceptions.RestClientException;
+import io.confluent.kafka.streams.serdes.avro.SpecificAvroSerde;
+import org.apache.kafka.clients.consumer.ConsumerConfig;
+import org.apache.kafka.streams.*;
+import org.apache.kafka.streams.kstream.*;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.time.Instant;
+import java.util.*;
+
+import static org.apache.kafka.common.utils.Utils.mkEntry;
+import static org.apache.kafka.common.utils.Utils.mkMap;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class FizzBuzzAggregatorTest {
+
+    private static TopologyTestDriver testDriver;
+    private static TestInputTopic<InputKey, Item> inputTopic;
+    private static TestOutputTopic<OutputKey, Output> outputTopic;
+    final private static String srConfigKey = "schema.registry.url";
```

Comment:
delete this `srConfigKey`

Reply:
I will put it in the setUp function, this config key is used many times in code.